vars/jobsParameters.groovy

```diff
@@ -111,6 +111,11 @@ static List devEnvironmentsList() {
             'volaris-2nd']
 }
 
+@NonCPS
+static List secretOperationsList () {
```

Comment:
As we plan in the future create jobs with similar parameters, better to name them more neutrally.